src/components/organisms/plans/plans.scss

```
@@ -0,0 +1,39 @@
+@use "../../variables" as *;
+.plans {
+  padding:56px 0 100px 0;
+  @media (max-width: $breakpoint-md) {
+    padding: 56px 0 35px 0;
+  }
+
+  .message {
+    padding-top: 0px;
+  }
+  .message .message-title {
+    text-align: center;
+    color: $orange_bistre;
+  }
+  .message .small {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+  }
```

```suggestion
.message {
  padding-top: 0px;
  
  .message-title {
    text-align: center;
    color: $orange_bistre;
  }
    
  .small {
    display: flex;
    justify-content: center;
    text-align: center;
  }
}
```
scssなのでこのように書くこともできます。